ntoskrnl/ke/amd64/context.c

```diff
@@ -292,6 +292,32 @@ KeTrapFrameToContext(IN PKTRAP_FRAME TrapFrame,
     if (OldIrql < APC_LEVEL) KeLowerIrql(OldIrql);
 }
 
+VOID
+RtlGetUnwindContext(
+    _Out_ PCONTEXT Context,
+    _In_ DWORD64 TargetFrame);
+
+VOID
+KiGetTrapContextInternal(
+    _In_ PKTRAP_FRAME TrapFrame,
+    _Out_ PCONTEXT Context)
+{
+    ULONG64 TargetFrame;
+
+    /* Capture the current context */
+    Context->ContextFlags = CONTEXT_FULL | CONTEXT_DEBUG_REGISTERS;
+    RtlCaptureContext(Context);
+
+    /* Get the volatile register context from the trap frame */
+    KeTrapFrameToContext(TrapFrame, NULL, Context);
+
+    /* The target frame is 0x80 bytes before the trap frame */
+    TargetFrame = (ULONG64)TrapFrame - 0x80;
```

Comment:
These 0x80 bytes appear to correspond also to the value of `MAX_SYSCALL_PARAM_SIZE` (which is used in `KiSetTrapContextInternal`). Do they have the same purpose in both functions?